WORKSPACE

```diff
@@ -35,6 +37,11 @@ nodejs_register_toolchains(
     node_version = "16.13.1",
 )
 
+nodejs_register_toolchains(
+    name = "node18",
+    node_version = "18.12.0",
```

Comment:
is this fine?

Reply:
I don't see what it's used for, removing for now